src/utils/apiEndpoints.ts

```diff
@@ -34,3 +34,20 @@ export const NetworkToChainId: Record<string, number> = {
   mainnet: 1,
   testnet: 2,
 };
+
+export function toNetwork(network: string): Network | undefined {
```

Comment:
is there a reason this needs to be a function and not enum?

Reply:
It was just that I needed string -> network, let me fix that